### DISCUSSION TITLE
configuring how deep the search should be

### DISCUSSION THREAD

**Discussion by alexisdal** [assafelovic/gpt-researcher#135] [Ideas]
Thanks a lot for gpt-search as it automates a lot of what I already do with chatgpt and plugins.

that said. I wish I could have a much deeper search done. Looking at the log file in the UI, it seems it made a few google searches and read a few pdf documents. Maybe 20 urls in total. 

for some topics, I am willing to mimick a user behavior doing the same for not with 20 urls, but more in the 500-1000 urls/pdf and the like. I understand the implications when it comes to API calls and subsequent costs with my openai key.

I wish I could find such a param in config.py

Can this be a feature request? 😉🙏

**Comment by assafelovic** [comment]
Hey @alexisdal , we just released a version that enables this now!
